beesdoo_shift/data/cron.xml

```diff
@@ -38,7 +38,7 @@
             <field name="name">Send weekly shift summary</field>
             <field name="model_id" ref="model_beesdoo_shift_shift" />
             <field name="state">code</field>
-            <field name="code">model._cron_send_weekly_emails()</field>
+            <field name="code">model._cron_send_weekly_emails(notice=1, period=7)</field>
             <field name="interval_number">7</field>
```

Comment:
I would add in the readme of the module that the interval number of the cron should be consistent with notice and period.